templates/prune-job.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+/usr/local/bin/tarsnap --fsck --keyfile "{{ item.keypath }}" \
+--cachedir "{{ item.cache }}" && \
+tarsnapper --config /etc/tarsnapper.yml  expire dalite-logs "{{ item.name }}"
```

Comment:
why is `dalite-logs` referenced here? It should already be handled by `item.name` no?